src-tauri/src/textures.rs

```diff
@@ -1,3 +1,6 @@
+// these commands are not yet used, allow dead code in this module
+#![allow(dead_code)]
+
```

Comment:
Perhaps this was a lazy choice, but they seem to be in development.

Reply:
yeah this file is still in a draft state so silencing them is totally acceptable